linker errors with Intel compiler

I see errors like
```
lib/libdeal_II.g.so.9.2.0-pre: error: undefined reference to 'void dealii::DoFTools::make_boundary_sparsity_pattern<...
```
I will investigate.

--- a/source/dofs/dof_tools_sparsity.inst.in
+++ b/source/dofs/dof_tools_sparsity.inst.in
@@ -145,7 +145,29 @@ for (scalar : COMPLEX_SCALARS; SP : SPARSITY_PATTERNS;
     // Function<deal_II_dimension+1,scalar>*> &boundary_ids, const
     // std::vector<types::global_dof_index>  &dof_to_boundary_mapping, SP
     // &sparsity);
+#endif
+#if deal_II_dimension == 1
+    template void DoFTools::make_boundary_sparsity_pattern<
+      hp::DoFHandler<deal_II_dimension, deal_II_dimension + 2>,
+      SP,
+      scalar>(
+      const hp::DoFHandler<deal_II_dimension, deal_II_dimension + 2> &dof,
+      const std::map<types::boundary_id,
+                     const Function<deal_II_dimension + 2, scalar> *>
+        &                                         boundary_ids,
+      const std::vector<types::global_dof_index> &dof_to_boundary_mapping,
+      SP &                                        sparsity);
 
+    template void DoFTools::make_boundary_sparsity_pattern<
+      DoFHandler<deal_II_dimension, deal_II_dimension + 2>,
+      SP,
+      scalar>(
+      const DoFHandler<deal_II_dimension, deal_II_dimension + 2> &dof,
+      const std::map<types::boundary_id,
+                     const Function<deal_II_dimension + 2, scalar> *>
+        &                                         boundary_ids,
+      const std::vector<types::global_dof_index> &dof_to_boundary_mapping,
+      SP &                                        sparsity);
 #endif
   }
 

